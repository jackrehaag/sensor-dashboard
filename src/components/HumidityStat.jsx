import React from "react";

const HumidityStat = ({ humidity }) => {
  return (
    <div className="teal current-temp statistic">
      <div className="value">{humidity}%</div>
      <div className="label">
        Humidity <i className="tint icon" />
      </div>
    </div>
  );
};

export default HumidityStat;
