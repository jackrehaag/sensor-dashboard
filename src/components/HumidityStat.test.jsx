import React from "react";
import HumidityStat from "./HumidityStat";
import renderer from "react-test-renderer";

test("it renders", () => {
  const component = renderer.create(<HumidityStat humidity={50} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
