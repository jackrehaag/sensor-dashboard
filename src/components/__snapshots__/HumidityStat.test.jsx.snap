// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="teal current-temp statistic"
>
  <div
    className="value"
  >
    50
    %
  </div>
  <div
    className="label"
  >
    Humidity 
    <i
      className="tint icon"
    />
  </div>
</div>
`;
